pyramid_rpc/jsonrpc.py

```diff
@@ -267,7 +267,7 @@ def text(self):
     phash = text
 
     def __call__(self, context, request):
-        return getattr(request, 'rpc_method') == self.method
+        return hasattr(request, 'rpc_method') and getattr(request, 'rpc_method') == self.method
```

Comment:
That would allow the predicate to succeed if self.method were None. I'm not sure if that's appropriate or not.